### ISSUE TITLE
uuid surpport

### ISSUE THREAD

**Issue by wolfwork** [Dasfaust/GlobalMarket#20]
Hi
First, i really appreciate you to make this plugin

How do  you add uuid support code?
I wish and waiting

**Comment by fatboy2380** [comment]
@Dasfaust is this any closer to becoming a reality? It's becoming an increasing issue with name changes loosing all data etc

**Comment by madtomic** [comment]
UUID is a most. Please consider to add this feature.

**Comment by Dasfaust** [comment]
It's nearing completion (version 2.0.0, that is. With UUID support). Most of the code is written and I just need to add features and test. I wanted to have it out way sooner but I've recently started a job and I'm still going to school full time, so it's been tricky to find time. Hopefully soon though.

**Comment by wolfwork** [comment]
wow!

**Comment by madtomic** [comment]
Any update or release date for 2.0? Will it support both 1.7.10 and 1.8.x?

**Comment by Dasfaust** [comment]
I have been working on it, it has entered the polishing phase. Making sure everything works, checking performance, cleaning up code, and re-adding small features. I have decided to go ahead and upload what I've got to a different branch so it can be tested and everyone can provide feedback. Once it's ready I'll merge it into the main branch and post a project update on BukkitDev/Spigot forums. Until then, it's considered beta and is pretty bare on features and documentation. Find out more [here](https://github.com/Dasfaust/GlobalMarket/tree/2.0.X#version-200-beta).
